integration_tests/persistent-workers-test/README.md

```diff
@@ -0,0 +1,191 @@
+# Remote Persistent Workers Integration Tests
+
+This directory contains comprehensive integration tests for the NativeLink Remote Persistent Workers feature.
+
+## Overview
+
+Remote Persistent Workers allow NativeLink to keep compiler processes hot by maintaining long-lived worker processes that can handle multiple compilation requests without the overhead of process creation for each action. This is particularly beneficial for JVM-based tools like `javac` and `scalac`.
+
+## Test Structure
+
+```
+persistent-workers-test/
+├── src/
+│   ├── persistent-worker-client.ts    # Client library for interacting with persistent workers
+│   ├── mock-scheduler-server.ts       # Mock scheduler for testing
+│   ├── persistent-worker.test.ts      # Unit and integration tests
+│   └── e2e.test.ts                    # End-to-end test suite
+├── package.json                       # Node.js dependencies
+├── tsconfig.json                      # TypeScript configuration
+└── jest.config.js                     # Jest test runner configuration
+```
+
+## Running Tests
+
+### Prerequisites
+
+```bash
+# Install dependencies
+npm install
+```
+
+### Unit Tests
+
+Run the Jest test suite:
+
+```bash
+npm test
+```
+
+With coverage:
+
+```bash
+npm test -- --coverage
+```
+
+Watch mode for development:
+
+```bash
+npm run test:watch
+```
+
+### Integration Tests
+
+Run the full integration test suite:
+
+```bash
+npm run test:integration
+```
+
+### End-to-End Tests
+
+Build and run E2E tests:
+
+```bash
+npm run build
+node dist/e2e.test.js
```

Comment:
This doesn't exist in `dist` after running build.

Reply:
This is not ready. I need to work on this more. Sorry for confusion